.github/workflows/publish-to-pypi.yaml

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the SG Otio project
+
+name: Publish to TestPyPi
+
+on:
+  release:
+    types: [published]
+# Keeping this around, useful when working on this, with just tags
+#  push:
+#    tags:
+#    - "*"
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    strategy:
+      matrix:
+        python-version: ['3.10']  # Not sure if we should build for Python 2?
```

Comment:
I think we should? Generally the more versions of python you support the better?